src/loop/pipes.mli

```diff
@@ -42,7 +42,7 @@ module Make
 
   type defs = [
     | `Type_def of Dolmen.Id.t * Expr.ty_var list * Expr.ty
-    | `Term_def of Dolmen.Id.t * Expr.ty_var list * Expr.term_var list * Expr.term
+    | `Term_def of Expr.term_const * Expr.ty_var list * Expr.term_var list * Expr.term
```

Comment:
I'd prefer if this was instead ``` `Term_def of Dolmen.Id.t * Expr.term_const```, this way the information about the dolmen identifier is not lost (keeping it will prove useful for the upcoming export function).

Reply:
That's a good point for ``` `Term_decl ``` , I'll have to think about it a bit more (in any case, as long as there's no release the change isn't really a problem).  So you can leave your change as-is.

The export function, or rather functionality is simply to have dolmen be able to output the typed problem in a chosen format (if possible, since not all formats support the same theories). I'm planning to add that soon-ish (I already have a big part of the code in archsat, which is capable of doing this kind of conversion).